x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/palette_preview.tsx

```diff
@@ -7,7 +7,7 @@
 
 import React from 'react';
 import { euiStyled } from '@kbn/kibana-react-plugin/common';
-import { InventoryColorPalette } from '../../../../../lib/lib';
+import type { InventoryColorPalette } from '../../../../../lib/lib';
```

Comment:
**nit:** just cause I noticed it in some other files and I see here the change is pretty specific, the `import type` syntax is useful to tell the bundler that if an import has runtime code but we just want to use it for typing (e.g. an `enum` or `class` have runtime code), we can use that syntax then to exclude from the bundling.
In this case `InventoryColorPalette` is a simple type with no runtime code, so the import can be the normal one as `import type` won't bring any benefit in this case.

Reply:
It doesn't hurt to keep anyway, it's non-blocking at all :)